src/leiningen/new/re_frame/project.clj

```diff
@@ -35,21 +36,33 @@
                                     "test/js"{{/test?}}{{#garden?}}
                                     "resources/public/css"{{/garden?}}]
 
-{{#garden?}}
+ {{#garden?}}
 
-  :garden {:builds [{:id           "screen"
-                     :source-paths ["src/clj"]
-                     :stylesheet   {{name}}.css/screen
-                     :compiler     {:output-to     "resources/public/css/screen.css"
-                                    :pretty-print? true}}]}
-{{/garden?}}{{#less?}}
+ :garden {:builds [{:id           "screen"
+                    :source-paths ["src/clj"]
+                    :stylesheet   {{name}}.css/screen
+                    :compiler     {:output-to     "resources/public/css/screen.css"
+                                   :pretty-print? true}}]}
+ {{/garden?}}{{#less?}}
   :less {:source-paths ["less"]
          :target-path  "resources/public/css"}
-{{/less?}}
-
-  :shell {:commands {"open" {:windows ["cmd" "/c" "start"]
-                             :macosx  "open"
-                             :linux   "xdg-open"}}}
+ {{/less?}{{#scss?}}
+  :scss {:builds
+         {:develop    {:source-dir "scss/"
+                       :dest-dir   "resources/public/css/"
+                       :executable "sassc"
+                       :args       ["-m" "-I" "scss/" "-t" "nested"]}
+          :production {:source-dir "scss/"
+                       :dest-dir   "resources/public/css/"
+                       :dest-dir   "public/css/"
+                       :executable "sassc"
+                       :args       ["-I" "scss/" "-t" "compressed"]
+                       :jar        true}}}}
```

Comment:
One too many closing `}`. Causes unmatched delimiter `}` syntax error when running the lein build (e.g. `lein dev`).